config/application.rb

```diff
@@ -175,6 +175,7 @@ class Application < Rails::Application
     # like if you have constraints or database-specific column types
     config.active_record.schema_format = :sql
 
+    config.active_record.yaml_column_permitted_classes = (config.active_record.yaml_column_permitted_classes + [Symbol]).uniq
```

Comment:
Perhaps add a comment linking to https://github.com/rails/rails/pull/45584? Although that PR was approved so perhaps we can wait a moment? There may be a new patch release soon.

Reply:
I thought so at first as well, that's why I've created two issues: for lock and unlock. Lock is temporary until new release and unlock just to not forget. In this case we would simply rely on Rails.

But after Evgeni's comment I thought that we could actually re-use that PR on "our terms". Just in case similar problem will occur in the future. Moreover, in this way we could allow more classes (if needed).

Reply:
After applying Rails' PR it throwed the same error but for `HashWithIndifferentAccess`, so I guess this approach is better than just relying on Rails...